website/docs/cloud-docs/workspaces/health.mdx

```diff
@@ -60,6 +60,12 @@ Terraform Cloud pauses health assessments if the latest run ended in an errored
 
 Health assessments may take longer to complete when you enable health assessments in many workspaces at once or your workspace contains a complex configuration with many resources.
 
+### On Demand Assessments
+
+If you are an administrator for the workspace, you should see a **Start health assessment** button on the Health pages with Ready status if you have all your [workspace requirements](/terraform/cloud-docs/workspaces/health#workspace-requirements) are satisfied. 
+
+  ![On Demand Assessments Screenshot: Copy generated token](/img/docs/on-demand-assessments.png)
```

Comment:
The screenshot is a great addition! It might be helpful to show the button within the full-page context.